examples/knowledge-base/lib/api.ts

```diff
@@ -1,5 +1,9 @@
 import fs from "node:fs/promises";
 import path from "node:path";
+import fetch from "isomorphic-unfetch";
+import dotenv from "dotenv-flow";
+
+dotenv.config();
```

Comment:
We only want to do this on local development.

Reply:
Actually I think we don't need this, as NextJs already handles the .env.local file.
https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

Reply:
Yeah but we need it for the script run by ts-node.